lib/src/component_common.dart

```diff
@@ -133,13 +133,4 @@ abstract class FluxComponentCommon<ActionsT, StoresT> extends react.Component
   Map<Store, StoreHandler> getStoreHandlers() {
     return {};
   }
-
-  /// Register a [subscription] that should be canceled when the component
-  /// unmounts. Cancellation will be handled automatically by
-  /// [componentWillUnmount].
-  @Deprecated(
-      'Subscriptions are now managed by the Disposable mixin. $v3Deprecation')
-  void addSubscription(StreamSubscription subscription) {
-    manageStreamSubscription(subscription);
```

Comment:
This is why I decided we needed a a v3. I'm not sure there's a way to keep this around but remove the `manageStreamSubscription` call as the replacement is `listenToStream` ... but we don't have a Stream, we only have the StreamSubscription.
So, I removed the whole addSubscription method, searched our sourcecode and recompiled our apps with this version of w_flux with the removals and it compiled without issue.
Is there another way to address the removal of manageStreamSubscription without cutting a V3 of w_flux?

Reply:
There is! We can use [getManagedDisposer](https://pub.dev/documentation/w_common/latest/disposable/Disposable/getManagedDisposer.html) to manage arbitrary objects:

```dart
  void addSubscription(StreamSubscription subscription) {
    getManagedDisposer(subscription.cancel);
  }
```

Reply:
Oh, perfect! I'll rework this to be a patch release then.